<?xml version="1.0" encoding="UTF-8"?> 
<!-- Launch file for running the delta-arm only !-->
<launch>
  <rosparam file="$(find delta_manipulator)/cfg/Hex_delta_config.yaml" />

  <node name="Example_trajectory_generator" pkg="delta_manipulator" type="Example_trajectory_generator.py" required="true" output="screen"/>

  <node name="Joystick_controller" pkg="delta_manipulator" type="Ground_based_joystick_control.py" required="true" output="screen"/>

  <node name="No_Stabilisation" pkg="delta_manipulator" type="No_Stabilisation.py" required="true" output="screen"/>
  
  <node name="Delta_inv_kinematics" pkg="delta_manipulator" type="Inverse_kinematics.py" required="true" output="screen"/>

  <node name="Servo_writer" pkg="delta_manipulator" type="Servo_writer.py" required="true" output="screen"/>

</launch>
